source/MaterialXCore/Document.h

```diff
@@ -25,6 +25,22 @@ using DocumentPtr = shared_ptr<class Document>;
 /// A shared pointer to a const Document
 using ConstDocumentPtr = shared_ptr<const class Document>;
 
+/// @class ReadOptions
+/// Based set of options for controlling the behavior of document read / import interface.
```

Comment:
Extending this to cover copyContentFrom() and similar use cases in the future, I would suggest:

"The base set of options for controlling the behavior of element read operations."